docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,75 +1,298 @@
+
+WARNING: Version {version} of {es} has not yet been released, so no
+Docker image is currently available for this version.
+
 endif::[]
+
 ifeval::["{release-state}"!="unreleased"]
-["source","yaml",subs="attributes"]
---------------------------------------------
-include::docker/docker-compose.yml[]
---------------------------------------------
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/elasticsearch/elasticsearch:{version}
+----
+
+endif::[]
+--
+
+. Pull the {kib} Docker image.
++
+--
+ifeval::["{release-state}"=="unreleased"]
+
+WARNING: Version {version} of {kib} has not yet been released, so no
+Docker image is currently available for this version.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/kibana/kibana:{version}
+----
+
+endif::[]
+--
+
+[[docker-start-enroll]]
+[discrete]
+==== Start a single-node {es} cluster and enroll {kib}
+
+The following command starts a single-node {es} cluster for development or
+testing.
+
+. Create an `elastic` network for your containers.
++
+[source,sh]
+----
+docker network create elastic
+----
+
+. Start {es} in Docker. A password is generated for the `elastic` user and
+output to the terminal, plus enrollment tokens for enrolling {kib} and adding
+additional nodes to your cluster.
++
+--
+ifeval::["{release-state}"=="unreleased"]
+
+WARNING: Version {version} of {es} has not yet been released, so no
+Docker image is currently available for this version.
+
 endif::[]
-// tag::docker-memory[]
-. Make sure Docker Engine is allotted at least 4GiB of memory.
-In Docker Desktop, you configure resource usage on the Advanced tab in Preference (macOS)
-or Settings (Windows).
-// end::docker-memory[]
 
-. Run `docker-compose` to bring up the three-node {es} cluster and {kib}:
+ifeval::["{release-state}"!="unreleased"]
+[source,sh,subs="attributes"]
+----
+docker run --name es-node01 --net elastic -p 9200:9200 -it docker.elastic.co/elasticsearch/elasticsearch:{version}
+----
+
+endif::[]
+--
++
+TIP: You might need to scroll back a bit in the terminal to view the password
+and enrollment token.
+
+. Copy the generated password and enrollment token and save them in a secure
+location. These values are shown only when you start {es} for the first time.
+
+. Locate the CA certificate in your container. This command returns the path
+to the `http_ca.crt` certificate, which you'll use in the next step.
 +
-["source","sh",subs="attributes"]
+[source,sh]
+----
+docker exec -it es-node01 /bin/bash -c "find /usr/share/elasticsearch -name http_ca.crt"
+----
+
+. Copy the `http_ca.crt` security certificate from your Docker container to
+your local machine. 
++
+[source,sh]
+----
+docker cp es-node01:/usr/share/elasticsearch/config/tls_auto_config_<timestamp>/http_ca.crt .
+----
+`<timestamp>`:: The timestamp of when the auto-configuration process created
+the security files directory in your Docker container.
+
```

Comment:
we will update that as we remove the timestamp from the dirname, but I haven't had the time to do this change yet. I'll make sure I'll adjust docs as part of that change

Reply:
We'll need to make that change across the docs where it's used, but will be an easy change.